storybook/stories/components/search/filters.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from 'react'
+import { Col, Grid } from 'reflex'
+import styles from '../../../shared/stories.css'
+
+export default function SearchFilters() {
+  return (
+    <Col size={3} className={styles.colors4} align="top">
```

Comment:
This size is specific to the search page, I think it would make more sense to define a single Filter than a sized column of filters. That way you can put the column with the rest of the layout but reduce the boilerplate